components/ui/AppCta.vue

```diff
@@ -37,5 +37,26 @@ export default class extends Vue {
   & :first-child {
     margin-right: $layout-02;
   }
+  svg {
+      overflow: visible;
+  }
+  &[target=_blank] {
+    path:nth-child(2) {
+      transform: translate(0, 0);
```

Comment:
Finally, move the styles inside each modifier block so the final arrangement should look like:
```
&_is-external {
  $arrow-path: "path:nth-child(2)";

  #{$arrow-path} {
    ...
  }

  &:hover #{$arrow-path} {
    ...
  }
}

&_is-internal {
  $arrow-path: "path:nth-child(1)";

  #{$arrow-path} {
    ...
  }

  &:hover #{$arrow-path} {
    ...
  }
}
```
This way, we follow BEM naming conventions. And we make a [legit use of nested selectors](https://en.bem.info/methodology/css/#nested-selectors) for the code we have no control over it. The new structure has the advantage of being more regular (revealing a further change for factoring out more common parts) and keeps all the behaviour related to internal/external animation inside the internal/external block modifiers.